Pages/Shared/_Layout.cshtml

```diff
@@ -44,13 +46,13 @@
 
     <footer class="border-top footer text-muted">
         <div class="container">
-            &copy; 2024 - XML_Project - <a asp-area="" asp-page="/Privacy">Privacy</a>
+            &copy; @DateTime.Now.Year - XmlProject - <a asp-area="" asp-page="/Privacy">Privacy</a>
```

Comment:
Replaced hardcoded year in the footer with a dynamic value.
Using @DateTime.Now.Year avoids the need for manual updates annually.

Reply:
Well, a legal consideration... 
Do you want to reflect that the content has been copywritten since a certain date?  Using the dynamic approach, it does not reflect that.  So, a user could argue that content previously captured was not copywritten.
Which isn't actually true, as content posted on the web is copyright, by default.  But adding the copyright notice does give you additional protection.